pkg/sentry/platform/ring0/entry_arm64.s

```diff
@@ -461,6 +461,14 @@ TEXT ·kernelExitToEl0(SB),NOSPLIT,$0
 	MOVD PTRACE_PSTATE(RSV_REG_APP), R1
 	WORD $0xd5184001  //MSR R1, SPSR_EL1
 
+	// need use kernel space address to excute below code, since
+	// after SWITCH_TO_APP_PAGETABLE the ASID is changed to app's
+	// ASID
+	WORD $0x10000061		//ADR R1, do_exit_to_el0
```

Comment:
Maybe we should add a space after '//', just like following.
// ADR R1, do_exit_to_el0

Reply:
done